### DISCUSSION TITLE
Tuba

### DISCUSSION THREAD

**Discussion by VarLad** [CodedOre/NewCaw#77] [General]
Found this today: https://tuba.geopjr.dev/
Looks pretty neat!
Just thought it'd be nice to bring to @IBBoard 's attention :P

**Comment by IBBoard** [comment]
I [posted about it the other day](https://github.com/CodedOre/NewCaw/pull/74#issuecomment-1480198265) (when it was still called Tooth) because they had taken a different approach to solving a problem that I was having.

I'm glad there are multiple clients. It gives people choice. Some apps are never going to be right for everyone. I went through Turpial, Polly and probably at least one more client before I found Corebird.

If Twitter hadn't closed the API then there would have been a clearer difference between the two clients, because Tootle/Tooth/Tuba was only ever trying to be a Fediverse client.

Was there anything specific that you wanted to discuss about Tuba and how it might influence NewCaw?

**Reply by VarLad** [reply]
Not really. Just thought it'd be interesting to bring it to your attention since both clients have similar goals (now that Twitter is a no-go).
I do think that two independent clients developing in parallel might be better since they can help solve some problems the other encounters
